documentation/writing_tasks.md

```diff
@@ -185,6 +185,18 @@ If your task must receive parameters only in a certain way, such as only
 `input_method`. For more information on `input_method`, see [Task metadata
 reference](#task-metadata-fields).
 
+### Task parameter constraints
+
+There are a couple important constraints to keep in mind when defining your
+task's parameters:
+
+- Task parameter names must begin with a lowercase letter, and may only include
```

Comment:
```suggestion
- Task parameter names must begin with a lowercase letter, and can only include
```
😁

Reply:
Noooo! 😂 I've been trying so hard not to use "may".

Reply:
You're doing great! This is kind of a tricky one cos you can't use might here. It's actually part of the reason we don't like"may" - the ambiguity for non-native speakers.

Reply:
Makes total sense 👍  Thanks Heston!